stmhal/boards/STM32F429DISC/mpconfigboard.h

```diff
@@ -29,8 +29,10 @@
 #define MICROPY_HW_UART2_PINS (GPIO_PIN_8 | GPIO_PIN_9)
 
 // I2C busses
-#define MICROPY_HW_I2C1_SCL (pin_A8)
-#define MICROPY_HW_I2C1_SDA (pin_C9)
+#define MICROPY_HW_I2C1_SCL (pin_B6)
+#define MICROPY_HW_I2C1_SDA (pin_B7)
```

Comment:
I'd probably remove I2C1 altogether, since B6/B7 are connected to the LCD, so I2C1 is essentially unusable on this board.